.vib/ejbca/cypress/cypress/plugins/index.js

```diff
@@ -0,0 +1,27 @@
+/// <reference types="cypress" />
```

Comment:
I think you can remove this file.

Reply:
No, I cannot as I am defining a custom `task` below. see line 21

Reply:
My bad then, forget I said anything 😁